ast/ddl_test.go

```diff
@@ -187,3 +187,7 @@ func (ts *testDDLSuite) TestColumnPositionRestore(c *C) {
 	}
 	RunNodeRestoreTest(c, testCases, "alter table t add column a varchar(255) %s", extractNodeFunc)
 }
+
+func (ts *testDDLSuite) TestCreateIndexStmtRestore(c *C) {
```

Comment:
we test `StmtNode` in `parser_test.go`, like this: #114

the cases of `CREATE INDEX` at `parser_test.go#L1781`

Reply:
remove this function please
rest LGTM

Reply:
remove this function please
rest LGTM